bentoml/_internal/frameworks/common/pytorch.py

```diff
@@ -89,14 +89,16 @@ def _run(
         params = Params(*args, **kwargs)
 
         def _mapping(
-            item: ext.PdDataFrame | ext.NpNDArray | torch.Tensor,
-        ) -> torch.Tensor:
+            item: ext.PdDataFrame | ext.NpNDArray | torch.Tensor | t.Any,
+        ) -> torch.Tensor | t.Any:
             if LazyType["ext.NpNDArray"]("numpy.ndarray").isinstance(item):
                 return torch.Tensor(item, device=self.device_id)
             if LazyType["ext.PdDataFrame"]("pandas.DataFrame").isinstance(item):
                 return torch.Tensor(item.to_numpy(), device=self.device_id)
-            else:
+            if LazyType["torch.Tensor"]("torch.Tensor").isinstance(item):
```

Comment:
Maybe discard these changes since it's in another pr?

Reply:
Let's just merge the other one, that should fix it.